modules/appnexusBidAdapter.js

```diff
@@ -292,7 +293,7 @@ export const spec = {
       serverResponse.tags.forEach(serverBid => {
         const rtbBid = getRtbBid(serverBid);
         if (rtbBid) {
-          const cpmCheck = (isAllowZeroCpmBidsEnabled(bidderRequest.bidderCode)) ? rtbBid.cpm >= 0 : rtbBid.cpm > 0;
+          const cpmCheck = (bidderSettings.get(bidderRequest.bidderCode, 'allowZeroCpmBids') === true) ? rtbBid.cpm >= 0 : rtbBid.cpm > 0;
```

Comment:
From readability perspective, to me, personally, previous version was much clearer.  
You just execute some function to get the info you need (`isAllowZeroCpmBidsEnabled`).  
With this new approach code is longer and it's more error prone since you have to remember name of the property `allowZeroCpmBids`.  

But, I guess since that is used in 2,3 places in the entire prebid.js than it's fine.

Reply:
I don't disagree but there's a problem with circular imports when we put everything in `utils`. (new `bidderSettings` needs `utils.deepClone`, and a few others)